eth/backend.go

```diff
@@ -538,7 +538,7 @@ func (s *Ethereum) Start() error {
 		maxPeers -= s.config.LightPeers
 	}
 	// Start the networking layer and the light server if requested
-	s.handler.Start(maxPeers)
+	//s.handler.Start(maxPeers)
```

Comment:
Comment out from L533 to L541.

Reply:
Code line from L533 to L539 is useless, close that is mutch better.